### ISSUE TITLE
Cannot Clone repo

### ISSUE THREAD

**Issue by robertwh09** [dotnet/samples#5519]
**Issue description**

This is probably my fault.  But why can't I clone or download the code from this repo.  No option is given on the web page and when i use the web URL in GitHub desktop it fails?  What am I doing wrong?  Screenshot attached.

[Screenshot 2022-11-26 134502](https://user-images.githubusercontent.com/47435375/204091989-60ffc399-d18a-4a25-98bb-1a6a125b5c95.png)

**Comment by Tigerhajen** [comment]
Yeah, what's up with this?

**Comment by robertwh09** [comment]
Thanks

**Comment by svick** [comment]
Have you tried cloning the whole repo and not just a single directory?

**Comment by robertwh09** [comment]
Yes I did that yesterday.  and it worked fine.  Was this working before and I just hadn't found it?
